### ISSUE TITLE
Loading Heroes

### ISSUE THREAD

**Issue by Vitae666** [MatthieuBonne/CQMacroCreator#8]
Since there's a "Save current Heroes to settings" button, can there be a "Load current Heroes from settings" button?
Basically, say I wanted to save 40 Heroes to always use in calcs, currently, CQMC loads a certain # of Heroes of which I have no idea where they come from and the # changes at times. I've started up CQMC and as many as 25 Heroes are the Current Best, and as low as 3. Like I said, no idea where those come from.
I wanted to check off my top 40 based on the Strength view in-game and save them and then when I wanted them again, load them up.

**Comment by MatthieuBonne** [comment]
I agree it's an interesting feature ! I'll think about it.

**Comment by Vitae666** [comment]
Kewl.
Only real reason I thought of it was cause I wanted to see what would happen if I used 40 Heroes to calculate against the Bosses. 
I'd added my top 40 STR Heroes (btw, ok, so, I guess CQCM takes the data from STR because the 20 that were loaded were the top 20 there) and my #40, Shaman, was actually used as the 3rd attacking Hero against MoaK. Shocked the hell outta me. Almost didn't add him.
If you don't addit, it's ok. Figured another way to do it. Just create a Bosses.txt file and change it up every now and then depending on how many boxes I open and auctions I win. Open cqinput file, paste the Bosses.txt data in there with the heroes/bosses/etc, and then just run CosmosQuest.exe

Side, but related, question: Like I'd mentioned, I'd had as few as 3 and as many as 25 Current Best Heroes. Any way to add a control to load up MORE Heroes easily? Drop down selector in increments of 5 up to whatever the hell the # causes CQ to crash? :) So far, 125 seems to do it for me. LOL
And then click a "Load them up" button that will just add up to that #? 
Not sure which would be easier. A load from file or a "rescan and add more" button.
As a user, the button would be far easier to use. Would allow better checking of ability to do Dungeons, Daily Quests, and Quests. But that's me being lazy :-)

Which leads me to ask, what exactly IS the point of saving the Heroes to the Settings.json file? It's not used for anything that I've been able to figure out. 
I added my Top 40 to CQMC and saved to Settings. Closed CQMC, reopened it, still have the original 20 load up. Check Settings and 40 are listed. Load up CQA. Click "Run Calc Now" thinking maybe it'll use the 40. Open gen.cqinput and only the 20 are listed. :-/
Not sure at all what saving the Heroes does then.
